users/tests/test_notifications.py

```diff
@@ -39,6 +43,35 @@ def notification_template_guardian_email_change_requested_fi():
     )
 
 
+@pytest.fixture
+def notification_template_user_auth_service_is_changing_fi():
+    return create_notification_template_in_language(
+        NotificationType.USER_AUTH_SERVICE_IS_CHANGING,
+        "fi",
+        subject="User authorization service is changing FI",
+        body_text="""Guardian FI: {{ guardian }}.
+        The change is happening {{ date_of_change_str|default('17.6.2024', true) }}.
+        {% if guardian.children.count() %}
+        Childrens' event participation history:
+
+        {% for child in guardian.children.all() %}
+        Child name: {{child.name}}
+
+        {% for enrolment in child.enrolments.all() %}
+        Event: {{enrolment.occurrence.event.name}}
+        Occurrence: {{enrolment.occurrence.time}}
+
+        {% endfor %}
+        {% endfor %}
+        {% endif %}
+        {% if children_event_history_markdown %}
+        Markdown: 
+        {{children_event_history_markdown}}
+        {% endif %}
+        """,  # noqa
```

Comment:
There is leading whitespace on all other lines except the first line, is that intentional? How about moving this to a variable/constant just before the pytest.fixture definition to keep it local but still make the indentation problems go away? How would you feel about this change?

Reply:
In a test it's not so important whether there's some extra whitespace there or not, so ok, just wanted to bring this up as it's easy to get more whitespace than one wants with the triple-quoted strings when inside deeper indentation levels.